### PR TITLE
CNV- 29935: RN static key

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.14

Issue: [CNV-29935](https://issues.redhat.com//browse/CNV-29935)

Link to docs preview: [RN: New](https://64575--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-new)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by lkladnit — APPROVED** [review]
/lgtm

**Review by skrthomas — COMMENTED** [review]
Lgtm. The only thing I’m wondering is if hot-plug needs a hyphen? I see it hyphenated in a comment in the file

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -182,7 +183,7 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-28944 Release note: Preview Cluster level eviction strategy change
 
 //CNV-29940 Release note: Preview UI Bridged network interface hot-plug for VMs
-* You can hot plug a bridge network interface and then restart or live migrate the virtual machine (VM) to apply the change. Hot unplugging is supported only for VMs created with {VirtProductName} 4.13 or later. 
+* You can hot plug a bridge network interface and then restart or live migrate the virtual machine (VM) to apply the change. Hot unplugging is supported only for VMs created with {VirtProductName} 4.13 or later.
```
[the remark this answers is not in the document]
Comment (skrthomas) [reply virt/release_notes/virt-4-14-release-notes.adoc:186]: Also is there a link associated with this feature?

[the remark this answers is not in the document]
Comment (apinnick) [reply virt/release_notes/virt-4-14-release-notes.adoc:186]: This is outside the scope of this PR.

**Comment by apinnick** [comment]
@skrthomas Is the peer review done?

**Review by lkladnit — APPROVED** [review]
/lgtm

**Pull request merged**